src/app/core/components/thread/thread.component.html

```diff
@@ -17,24 +17,66 @@
             *ngFor="let event of state.data"
           ></alg-thread-message>
         </ng-container>
-        <form class="footer-panel" [formGroup]="form" (ngSubmit)="sendMessage()">
-          <textarea
-            class="textarea"
-            placeholder="Send a message..."
-            pInputTextarea
-            formControlName="messageToSend"
-            [rows]="1"
-            [autoResize]="true"
-            (keydown.enter)="$event.preventDefault(); sendMessage()"
-          ></textarea>
-          <button
-            class="alg-button-icon primary-color rounded send-button"
-            type="submit"
-            icon="pi pi-send"
-            pButton
-            [disabled]="!form.get('messageToSend')?.value?.trim()"
-          ></button>
-        </form>
+        <div class="footer-panel" *ngrxLet="isThreadStatusOpened$; let isThreadStatusOpened">
+          <form class="send-form" [formGroup]="form" (ngSubmit)="sendMessage()">
+            <textarea
+              class="textarea"
+              placeholder="Send a message..."
+              pInputTextarea
+              formControlName="messageToSend"
+              [rows]="1"
+              [autoResize]="true"
+              (keydown.enter)="$event.preventDefault(); sendMessage()"
+            ></textarea>
+            <button
+              class="alg-button-icon primary-color rounded send-button"
+              type="submit"
+              icon="pi pi-send"
+              pButton
+              [disabled]="!form.get('messageToSend')?.value?.trim() || !isThreadStatusOpened"
+            ></button>
+          </form>
+          <div class="thread-status">
+            <div class="thread-status-caption">
+              <ng-container *ngIf="isThreadStatusOpened; else notOpenedStatusCaption">
+                <span i18n>This thread is open</span>
+              </ng-container>
+              <ng-template #notOpenedStatusCaption>
+                <span i18n>This thread is closed</span>
+              </ng-template>
+            </div>
+            <ng-container *ngrxLet="isCurrentUserThreadParticipant$; let isCurrentUserThreadParticipant">
+              <ng-container *ngrxLet="allowToOpenThreadState$; let allowToOpenThreadState">
```

Comment:
If you have 2 info to unwrap you can just do something like `*ngrxLet="{ isCurrentUserThreadParticipant: isCurrentUserThreadParticipant$, allowToOpenThreadState: allowToOpenThreadState$ } as threadInfo"` instead of cascading containers.

Reply:
Agree, I didn't found this in doc before, but seems I've seen it in our project